types/stripe/index.d.ts

```diff
@@ -4120,6 +4120,8 @@ declare namespace Stripe {
              */
             currency: string;
 
+            customer?: string;
```

Comment:
This field is mentioned in the stripe API docs without explanation. In practice, it does appear to be the customer ID.
https://stripe.com/docs/api/customer_bank_accounts/object#customer_bank_account_object-customer